pkg/ccl/utilccl/licenseccl/license.proto

```diff
@@ -37,4 +39,7 @@ message License {
     }
 
     Usage usage = 5;
+
+    bytes license_id = 6;
```

Comment:
it might be helpful to add comments here that these are UUIDs. (if i remember correctly.)

Reply:
If using `bytes` to represent a UUID is atypical, (it is in managed-service at least), It would also be nice to document the reason that `bytes` was used rather than the typical type.  (string in the case of managed-service)